src/main/java/service/activities/Purchase.java

```diff
@@ -0,0 +1,24 @@
+package service.activities;
+
+import db.Storage;
+import javax.management.openmbean.InvalidKeyException;
+import service.fruitrecord.FruitRecord;
+
+public class Purchase implements ActivityHandler {
```

Comment:
Same for all handlers
```suggestion
public class PurchaseHandler implements ActivityHandler {
```